fusillade-api.yml

```diff
@@ -935,6 +935,109 @@ paths:
           description: Unexpected error.
       tags:
         - role
+  /v1/resource:
+    get:
+      summary:  List avaialable resource types
+      description: List avaialable resource types
+      operationId: fusillade.api.resource.get
+      security:
+        - BearerAuth: []
+      parameters:
+        - $ref: '#/components/parameters/next_token'
+        - $ref: '#/components/parameters/per_page'
+      responses:
+        200:
+          $ref: '#/components/responses/200_paged'
+        206:
+          $ref: '#/components/responses/206'
+        401:
+          $ref: '#/components/responses/401'
+        403:
+          $ref: '#/components/responses/403'
+        default:
```

Comment:
Should 500, 502, 503, 504 errors be added?

Reply:
Right now only 500 is returned for uncaught errors. I can add better error messages in the future as needed.

Reply:
do you think it's needed with the addition of the API, and should It be apart of all of the any other APIs?